Yoast/wordpress-seo
# Simplify webpack config

## Summary

This PR can be summarized in the following changelog entry:

* _not applicable_

## Relevant technical choices:

* The dev and prod files weren't doing a lot of changes so it is clearer
to have these changes inline.

## Test instructions

This PR can be tested by following these steps:

* Make sure `grunt webpack:buildDev`, `grunt webpack:buildProd`, `yarn start` and `yarn build` still work as expected.

## Quality assurance
